lib/model/sign_up.rb

```diff
@@ -0,0 +1,23 @@
+class SignUp < Sequel::Model(:userdetails)
+  dataset_module do
```

Comment:
What does ` dataset_module ` add?  Not sure I understand it.  I like this refactoring though.

Reply:
I found a nice cheat sheet here:

https://gist.github.com/odigity/2d8123a90e8c2331d0ad#file-sequel_scopes-rb-L5